src/parser/wsdl.js

```diff
@@ -104,8 +104,13 @@ class WSDL {
         self.xmlnsInEnvelope = self._xmlnsMap();
         callback(err, self);
       });
+    }
 
-    });
+    if (self.options.forceSyncLoad) {
+      loadUpSchemas()
```

Comment:
I don't think `loadUpSchemas` is completely synchronous as it calls `processIncludes`, which is async. Do you want to propagate the flag into `processIncludes` too?

Typically, we would like to have `load` and `loadSync` methods instead of using an option.

Reply:
I'll update to propogate the forceSyncLoad option now...

But could rewrite on Monday to use a different method and add a flag to the load recursive methods if that is your preference... 

Does the following look ok? with a new _loadInternal function which would take a new boolean `sync` parameter? (_loadInternal(includePath, options, sync, function(err, wsdl) {)

WSDL.loadSync(includePath, options, function(err, wsdl) {
which would call 
_loadInternal(includePath, options, true, function(err, wsdl) {

original load
WSDL.load(includePath, options, function(err, wsdl) {
_loadInternal(includePath, options, false, function(err, wsdl) {